cocos/scripting/js-bindings/manual/ScriptingCore.cpp

```diff
@@ -396,17 +396,7 @@ bool JSB_core_restartVM(JSContext *cx, uint32_t argc, jsval *vp)
 
 bool JSB_closeWindow(JSContext *cx, uint32_t argc, jsval *vp)
 {
-    EventListenerCustom* _event = Director::getInstance()->getEventDispatcher()->addCustomEventListener(Director::EVENT_AFTER_DRAW, [&](EventCustom *event) {
-        Director::getInstance()->getEventDispatcher()->removeEventListener(_event);
-        CC_SAFE_RELEASE(_event);
-        
-        ScriptingCore::getInstance()->cleanup();
```

Comment:
no meaning to do `cleanup` in `JSB_closeWindow `, `Director::getInstance()->end()` will do it latterly. another reason is `_event ` have been freed when the lambda callback trigger